server/routes/orion.js

```diff
@@ -163,10 +164,9 @@ exports.readRestaurantWithDate = function(req, res) {
       actualOccupancyLevels = utils.getOccupancyLevels(
         restaurantReservations,
         req.params.id);
-      console.log(actualOccupancyLevels);
       occupancyLevelsObject = utils.updateOccupancyLevels(
         actualOccupancyLevels, req.params.date);
-      utils.getListByType('Restaurant', req.params.id, req.headers)
+      utils.getListByType('Restaurant', restaurantId, req.headers)
```

Comment:
I think it could be too late but indenting every nested promise it is unnatural, because promises were invented just for avoiding nesting of nesting

```js
a.then(function(x) {
  return b();
}.then(function(y) {
   doWhatever();
})
```